.eslintrc.js

```diff
@@ -21,6 +21,6 @@ module.exports = {
     '@typescript-eslint/explicit-function-return-type': 'off',
     '@typescript-eslint/explicit-module-boundary-types': 'off',
     '@typescript-eslint/no-explicit-any': 'off',
-    "max-len": ["error", { code: 80, ignoreComments: true }],
+    "max-len": ["error", { code: 100, ignoreComments: true }],
```

Comment:
I'd put this in other PR but it's not a blocker so far

Reply:
Additionally, just curious, was this change a decision that you all agree as a team? 👀